pkg/server/server.go

```diff
@@ -402,3 +403,39 @@ func (s *DriverServiceServer) Monitoring(ctx context.Context, req *pb.Monitoring
 	log.Info("Monitoring Routine Done", zap.String("sp", sp.GetUuid()))
 	return &pb.MonitoringResponse{}, nil
 }
+
+func (s *DriverServiceServer) SuspendMonitoring(ctx context.Context, req *pb.MonitoringRequest) (*pb.MonitoringResponse, error) {
+	log := s.log.Named("Monitoring")
+	sp := req.GetServicesProvider()
+	log.Info("Starting Monitoring Routine", zap.String("sp", sp.GetUuid()))
+
+	c, err := one.NewClientFromSP(sp, log)
+	if err != nil {
+		return nil, status.Errorf(codes.InvalidArgument, "Error making client: %v", err)
+	}
+
+	for _, group := range req.GetGroups() {
+		for _, inst := range group.GetInstances() {
+			vmid, err := one.GetVMIDFromData(c, inst)
+			if err != nil {
+				log.Sugar().Errorf("VM id obtaining failed for id=%s", inst.Uuid)
```

Comment:
1. Wrong Log format, standard zap.Fields functionality must be used, log level must be WARN
2. no `continue` statement, will cause errors down the loop